src/main/java/frc/robot/subsystems/climber/ClimberSubsystem.java

```diff
@@ -30,6 +32,30 @@ public Command manualMoveBackwardCommand() {
         );
     }
 
+    public Command climbSequenceCommand() {
+        return m_deployed ? liftRobotCommand() : deployClimberCommand();
+    }
+
+    public Command deployClimberCommand() {
+        return new FunctionalCommand(
+            () -> m_deployed = true,
```

Comment:
I would set m_deployed based on whether the arm is actually deployed, not based on whether we've run the command.